### ISSUE TITLE
Windows Installs PS4 controller drivers on top of SCP software, and ruins it completely. FIXED MYSELF SORRY BOUT THAT

### ISSUE THREAD

**Issue by ChosenJames** [nefarius/ScpToolkit#738]
FINAL EDIT: I fixed my issue by bumming around the device manager and frantically doing everything with both said programs and solved my own issue within 2 hours after this post was made, sorry now I feel kinda stupid, but feel free to read it all and ask question if you have a similar problem that I had I might get more detail into what I did to fix this issue for me.

Hey guys, I recently upgraded to windows 10 Pro, and when I plugged the controller in with the SCP service active it ran perfectly fine.. until windows notifies me that it is currently installing the controller. I panic and unplug and replug it in after it finished and to my un-surprise, BAM, the controller no longer works and the service takes up 30% CPU usage and no longer functions.

This is a different-ish version of the software that uses zadig, so I went here to get the toolkit, and when I installed it and went to install drivers, it asked to select a controller to start driver installation, that's the thing.. this program won't recognize that there's a controller plugged in. I checked device manager and it is recognized and the windows test controller function through use it seems it works perfectly. So I did the ToolKit driver cleaner file and it said it cleared everything, but the controller still isn't recognized.. I tried to uninstall the ps4 controller completely from the PC but there is no information on how to do so, and i've looked and tried, there is no option to uninstall drivers, only unplugging them from the PC.

The only fix I know that WILL work is wiping my PC and I would prefer not to do that at the moment..
I had this similar problem when I had windows 8.1 PRO and got a new PS4 controller and it installed those drivers on top of SCP just like now, but I fixed it by switching usb ports, I've tried all 2.0 and 3.0 ports and that doesn't work for me now.

So please I would provide any information to kind of look for a fix, if I absolutely HAVE to, I will look into wiping the computer just to get my controller functioning again.. 

DAMN YOU WINDOWS 10 UPGRADE! Jokes aside, thanks for reading and please, help me :)

**Comment by ChosenJames** [comment]
Addition: I fixed the actual controller by replacing the system drivers with zadig and the controller now functions, but now I have the issue of the wireless controller and the xbox 360 controller being active at the same time, causing my CPU usage to continue being 30% at all times when plugged in, when not plugged in, it is 0% as normal. I am still scrambling around trying to fix this high CPU usage, I've fixed it before but now I am trying everything and I am really starting to thin out of idea's at the moment.

The 30% CPU usage is with the older zadig version, and the newest beta SCPToolKit as well, in fact, I can't get the controller to be detected still with the new SCPToolKit.

**Comment by ChosenJames** [comment]
I solved the CPU usage, it was related to having two wireless controllers installed because SCP installed another one on top of the one that pre-existed before installing the drivers.

I went into the Device manager and noticed the two wireless controllers, I Uninstalled one of them and I of course had a tick box saying delete drivers, I did so and it prompted me to restart my PC, and so far the CPU usage has dropped back down to 0.2-1% usage as normal! I'll leave this hear so that anyone else maybe having this issue can come across it maybe and it may give them a lead to fix the High CPU usage.

Also, I find that I feel kind of stupid that I made a post for help and then solve it long before anyone would ever read this. xD
